### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ulugbekna**
Exceedingly happy to have this PR. Should make users' lives easier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,21 +197,56 @@ to re-open the REPL to change it.
...
 ## Commands
 
-You can execute it by entering the following command at the command palette
-<kbd>Ctrl</kbd>+<kbd>P</kbd> (<kbd>Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> on
-MacOS).
+An easy way to see what commands are offered by the extension in the currently
+open file, you can invoke VSCode Command Palette and search for commands with
+prefix `OCaml:`:
+
+![commands](./readme-assets/commands.png)
```
Comment (mnxn) [inline README.md:204]: Broken path. Should be `./assets`?

**Review by mnxn — CHANGES_REQUESTED** [review]
`ext-in-marketplace.png` doesn't seem to be used anywhere.

File: README.md (hunk 1 above)
Comment (mnxn) [reply README.md:204]: Same thing with the `![pick sandbox]` image.

Comment (ulugbekna) [reply README.md:211]: strange, the markdown seems rendered fine here: https://github.com/ulugbekna/vscode-ocaml-platform/tree/update-readme

**Review by mnxn — APPROVED** [review]
Looks great!

**Pull request merged**